### PR TITLE
Add return type for CancelMerchOrder

### PR DESCRIPTION
**Pull request by maxwn04**
# Info

Closes **#288**. (If there is no issue for this pull request yet, please create one or
delete this line if the pull request is for a very minor tweak).

# Description

What changes did you make? List all distinct problems that this PR addresses. Explain any relevant
motivation or context.

Adds missing return type

## Changes

- Adds a api response type for cancel merch order

# Type of Change

- [X] Patch (non-breaking change/bugfix)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (A change to a README/description)
- [ ] Continuous Integration/DevOps Change (Related to deployment steps, continuous integration
      workflows, linting, etc.)
- [ ] Other: (Fill In) 

If you've selected Patch, Minor, or Major as your change type, **make sure to bump the version before merging in `package.json`!**
# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally.
- [ ] on the testing API/testing database.
- [ ] with appropriate Postman routes. Screenshots are included below.

# Checklist

- [x] I have performed a self-review of my own code.
- [x] I have followed the style guidelines of this project.
- [ ] I have appropriately edited the API version in the `package.json` file.
- [x] My changes produce no new warnings.

# Screenshots

Please include a screenshot of your Postman testing passing successfully.

### CODE REVIEW

**Comment by maxwn04** [comment]
image
Cancel Merch Order Postman

File: types/ApiResponses.ts [hunk 1]
```diff
@@ -452,3 +452,7 @@ export interface PatchResumeResponse extends ApiResponse {
 }
 
 export interface DeleteResumeResponse extends ApiResponse {}
+
+export interface CancelMerchOrderResponse extends ApiResponse {
```
Comment (nik-dange) [inline types/ApiResponses.ts:456]: Can this (and possibly any other Merch Store related return types be moved to the correct part of this file?

**Review by nik-dange — CHANGES_REQUESTED** [review]
one nit but lgtm

**Comment by nik-dange** [comment]
Also make sure to bump package.json version

File: types/ApiResponses.ts (hunk 1 above)
Comment (nik-dange) [reply types/ApiResponses.ts:456]: Feel free to clean this file up if you want it feels very messy at the bottom

**Review by dowhep — APPROVED** [review]
Great Catch with the missing return type information!

LGTM 🚀 🚀 

As Nikhil noted, make sure to bump package.json version before merging

**Review by nik-dange — APPROVED** [review]

**Pull request merged**